x/mongo/driver/operation.go

```diff
@@ -767,15 +767,19 @@ func (op Operation) createQueryWireMessage(dst []byte, desc description.Selected
 		return dst, info, err
 	}
 
+	// Add server API information before calling addSession() because it will modify
+	// transaction.state.
```

Comment:
Is the period between "transaction" and "state" intentional?

Reply:
Nope! Changed to a space.